### ISSUE TITLE
Tab Control in Pro Version - IconTextSelected never displays

### ISSUE THREAD

**Issue by rossvernal** [UXDivers/Grial-UI-Kit-Support#518]
When using a TabControl, I can always see the IconText icon that I've setup. When clicking on the tab however, the IconTextSelected state is never displayed

### Description
Using this code,  I can replicate it on Xamarin forms in iOS and Android. Xamarin forms version is 3.4.0.1009999.

### Steps to Reproduce
```
<tab:TabControl
			Grid.Row="1"
            TabStripBackgroundColor="{ DynamicResource AccentColor }"
			TabStripPlacement="Bottom"
			StyleClass="CustomTabControlStyle"
			SelectedIndex="0" VerticalOptions="FillAndExpand">
			
			
			<tab:TabItem
				IconText="{ x:Static local:IcoMoonFont.HomeOutline }" 
                IconTextSelected="{ x:Static local:IcoMoonFont.Home }" >

                <ListView
		            SeparatorVisibility="None"
		            SeparatorColor="{ DynamicResource ListViewSeparatorColor }"
		            ItemsSource="{Binding xxxxxxx }"
		            RowHeight="400"
		            ItemTapped="OnItemTapped"
                        x:Name="ExploreListView"
		            HasUnevenRows="false" IsPullToRefreshEnabled="True">        
                    <ListView.ItemTemplate>
                        <DataTemplate>
                            <ViewCell>
                                <localTemplate:ArticleItemTemplate />
                            </ViewCell>
                        </DataTemplate>
                    </ListView.ItemTemplate>
                </ListView>

            </tab:TabItem>
		
			
            <tab:TabItem 
				IconText="{ x:Static local:IcoMoonFont.EyeOutline }" 
                IconTextSelected="{ x:Static local:IcoMoonFont.Eye }">
                
                <ListView
		SeparatorVisibility="None"
		SeparatorColor="{ DynamicResource ListViewSeparatorColor }"
		ItemsSource="{Binding xxxxxxxxxx}"
		RowHeight="400"
		ItemTapped="OnItemTapped"
            x:Name="LookoutListView"
		HasUnevenRows="false" IsPullToRefreshEnabled="True">
                    <ListView.ItemTemplate>
                        <DataTemplate>
                            <ViewCell>
                                <localTemplate:ArticleItemTemplate />
                            </ViewCell>
                        </DataTemplate>
                    </ListView.ItemTemplate>
                </ListView>

            </tab:TabItem> 
```
### Expected Behavior
Different icon should show once the tab has been selected.

### Actual Behavior
there is no change to the icon

### Basic Information

- Version with issue: Forms 3.4.0.1009999, Grial UXDivers.Artina.Controls.Tab v2.7.5
- Last known good version: unknown
- IDE: VS2017 Commumunity
- Platform Target Frameworks: 
  - iOS:  12.1
  - Android: 9.0 
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices: All

**Comment by jpbrocca** [comment]
Hi @rossvernal, sorry for the delay on this.

To make it work you will need to change the current `Label` used to display the icons in the `CustomDefaultTabItemTemplate` (inside your "`TabControlCustomResources.xaml`" file):

~~~
<!--FONT ICONS-->
<Label                         
    x:Name="IconTextLabel"
    IsVisible="{ Binding Text, Source={x:Reference IconTextLabel}, Converter={ StaticResource IsNotNullConverter } }"
    Text="{ Binding IconText }"
    Style="{ Binding IsSelected, Converter={ StaticResource CustomTabItemIconTextStateStyleConverter }, Mode=OneWay }"
>
    <Label.Triggers>
        <MultiTrigger TargetType="Label">
            <MultiTrigger.Conditions>
                <BindingCondition Binding="{ Binding IsSelected }" Value="true" />
                <BindingCondition Binding="{ Binding IconTextSelected, Converter={ StaticResource IsNotNullConverter }}" Value="true" />
            </MultiTrigger.Conditions>          
            
            <Setter Property="Text" Value="{ Binding IconTextSelected }" />
        </MultiTrigger>
    </Label.Triggers>
</Label>
~~~

Basically, the multi trigger will do the trick :)

Let me know if you have questions.

Best regards,
JP

**Comment by rossvernal** [comment]
@jpbrocca hey - that worked, had a little trouble tracking it down because our version doesn't have 

`x:Name="IconTextLabel"`

but once I found it and applied, it all works perfectly.

Thank you so much for helping me to fix it!
